benchmarks/nemo/nemolite2d/manual_versions/psykal_opencl/nemolite2d.f90

```diff
@@ -44,7 +44,6 @@ program gocean2d
   ! points immediately to the North and East of a T point all have the
   ! same i,j index).  This is the same offset scheme as used by NEMO.
   model_grid = grid_type(GO_ARAKAWA_C, &
-  !  BC_PERIODIC, BC_NON_PERIODIC ??
                          (/GO_BC_EXTERNAL,GO_BC_EXTERNAL,GO_BC_NONE/), &
```

Comment:
Please add an attribution to Hedong Liu, NOC in the model docstring above (line 12).

Reply:
Done